src/gpu/nvidia/cudnn_convolution_impl.hpp

```diff
@@ -495,6 +549,11 @@ struct cudnn_convolution_impl_fwd_t : public cudnn_convolution_impl_base_t {
             weights = w_scratch;
         }
 
+        float *y_fp32_data = nullptr;
+        if ((dst_scale || src_scale) && data_types[io::y] == CUDNN_DATA_INT8) {
```

Comment:
To simplify reading, avoid bugs and to remove copy-paste I would recommend to create a method for the following and reuse it everywhere:
`bool y_f32_is_required() const { return (do_src_scaling || do_dst_scaling || do _wei_scaling) && data_types[io::y] == CUDNN_DATA_INT8; }`

Reply:
I added the new function and is being called where needed. If you find some additional place to use it please let me know. Thanks.